kong/plugins/rate-limiting/policies/init.lua

```diff
@@ -108,27 +109,35 @@ return {
             ngx_log(ngx.ERR, "failed to query Redis: ", err)
             return nil, err
           end
-
-          red:init_pipeline((not exists or exists == 0) and 2 or 1)
-          red:incrby(cache_key, value)
           if not exists or exists == 0 then
-            red:expire(cache_key, EXPIRATIONS[period])
-          end
-
-          local _, err = red:commit_pipeline()
-          if err then
-            ngx_log(ngx.ERR, "failed to commit pipeline in Redis: ", err)
-            return nil, err
+            t[#t + 1] = { key = cache_key, expires = EXPIRATIONS[period]}
+          else
+            t[#t + 1] = { key = cache_key }
           end
         end
       end
+      red:init_pipeline()
+      for i = 1,#t,1 do
+        red:incrby(t[i].key, value)
+        ngx.log(ngx.ERR,t[i].key)
```

Comment:
Please remove those error logging statements as they aren't logging anything particularly useful to the user (and aren't logging an error anyways).